packages/typescript/src/index.ts

```diff
@@ -51,7 +51,7 @@ export default function typescript(options: RollupTypescriptOptions = {}): Plugi
           formatHost,
           resolveModule,
           parsedOptions,
-          writeFile(fileName, data) {
+          writeFile(fileName: string, data: string) {
```

Comment:
These type annotations shouldn't be necessary as they're provided by the `createWatchProgram` function. Same goes for `parsedOptions.fileNames.forEach`.

Reply:
You're right. It was because of the way the TypeScript types were being imported. I'm not quite sure why it was being done the way it was. I've gone ahead and updated it, but let me know if there's a reason to leave it as it was and I can revert it